fixtures/button.fixture.js

```diff
@@ -0,0 +1,34 @@
+import Button from "@khanacademy/wonder-blocks-button";
+
+import {describe, expect, test, render, simulate} from "./helpers.js";
+
+describe("Button", () => {
+    test("primary", async () => {
+        const element = await render(<Button>Hello, world!</Button>);
+        await expect(element).toMatchScreenshot();
+        await simulate({type: "mousemove", clientX: 50, clientY: 25});
+        await expect(element).toMatchScreenshot({outset: 4});
+        await simulate({type: "mousedown"});
+        await expect(element).toMatchScreenshot();
+    });
+
+    test("primary (tab to focus)", async () => {
+        const element = await render(<Button>Hello, world!</Button>);
+        await simulate({type: "keypress", key: "tab"});
```

Comment:
@matchu we should be able to write a test for focus capture using this.  I'll try to write a test for this.

Reply:
Ooh yay that would be so exciting!

Reply:
I did write a test for this.  Check out modal.fixture.js down below.